pkg/provider/provider.go

```diff
@@ -72,9 +81,8 @@ func Provider() *schema.Provider {
 }
 
 func ConfigureProvider(s *schema.ResourceData) (interface{}, error) {
-	dsn, err := DSN(s)
+	dsn, err := DSN(s) // got an error
```

Comment:
what's this `//got an error` comment?

Reply:
oh, this was when I was trying to print something. The comment is outdated